composer.json

```diff
@@ -6,21 +6,21 @@
         "smarty/smarty" : "~3.1.33",
         "firebase/php-jwt" : "~3.0",
         "google/recaptcha": "~1.1",
-        "php-http/guzzle6-adapter": ">=2.0.0",
+        "php-http/guzzle7-adapter": "0.1.1",
```

Comment:
php-http/guzzle6-adapter doesn't support PHP 8.0.0 but the GitHub repo advises to use php-http/guzzle7-adapter instead.